spec/tic_tac_toe_spec.rb

```diff
@@ -1,7 +1,49 @@
+require 'game'
+require 'output'
+require 'input'
+
+
 RSpec.describe TicTacToe do
 
-  it "asserts true equsls true" do
-    expect(true). to eq(true)
+  before(:each) do
+    @game = Game.new
+    @output = Output.new
```

Comment:
Here is the opportunity to create a MockOutput class that can track calls instead of actually printing to standard out.